LoopUI/Views/SettingsView.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  SettingsView.swift
+//  LoopUI
+//
+//  Created by Rick Pasetto on 6/24/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import LoopKitUI
+import SwiftUI
+
+public struct SettingsView: View, HorizontalSizeClassOverride {
+    @Environment(\.dismiss) var dismiss
+    
+    @ObservedObject var viewModel: SettingsViewModel
+
+    public init(viewModel: SettingsViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    public var body: some View {
+        NavigationView {
+            List {
+                loopSection
+                if viewModel.showWarning {
+                    alertPermissionsSection
+                }
+                therapySettingsSection
+                deviceSettingsSection
+                supportSection
+            }
+            .listStyle(GroupedListStyle())
+            .navigationBarTitle(Text(LocalizedString("Settings", comment: "Settings screen title")))
+            .navigationBarBackButtonHidden(false)
+            .navigationBarHidden(false)
```

Comment:
[nit] Is this redundant, or is it serving some purpose I don't know about?

Reply:
I don't think so...one says we want the navigationBar to show, the other is to make sure we want the Back button to show...

Reply:
I think they're false by default, which is why I asked!